app/views/devise/invitations/edit.html.erb

```diff
@@ -2,7 +2,9 @@
 <div class="crayons-layout crayons-layout--limited mb-10" style="max-width:640px">
   <div class="crayons-notice mt-4">
     <h1>Welcome to <%= community_name %></h1>
-    <h3><%= SiteConfig.tagline %></h3>
+    <% if SiteConfig.tagline.present? %>
```

Comment:
**suggestion (non-blocking):** Including this addition, I think we have this conditional in 5 places now. maybe we could move it into the `ApplicationHelper`?

```ruby
def tagline
  return if SiteConfig.tagline.blank?

  content_tag(:h3, SiteConfig.tagline)
end
```

and then use 

```ruby
<%= tagline %>
```

ERb deals gracefully with `nil`s:

```ruby
template = ERB.new("<%= nil %>")
template.result
#=> ""
```

(The result for the `if` version would be `" \n"`)

Reply:
Hm, I might end up doing this in a separate PR since I'll be making some more changes in this portion of the codebase to handle https://github.com/forem/InternalProjectPlanning/issues/87 and https://github.com/forem/InternalProjectPlanning/issues/89...

Since this is a non-blocking suggestions, I'm going to come back to cleaning this (and maybe some more conditional?) up when I finish up those other two issues :)

Reply:
That sounds perfect, totally just a suggestion and it can most definitely wait/be done in a different way/ignored.